meson: command not found (nixos)

## Description
When trying to run `nix-build`, NixOS is throwing an error about missing command `meson`. I have meson `1.7.2` installed on system.


## Steps to Reproduce
1. `git clone https://github.com/trip-zip/somewm.git`
2. `cd <somewm-repo>`
3. `nix-build`

## Expected Behavior
Should build somewm successfully. No result folder created due to error.


## Actual Behavior
Failing to build due to supposedly missing `meson` command.


## Version Info
Binary doesn't install. N/A


## Configuration
<!-- If relevant, include the parts of your rc.lua that trigger the bug -->

N/A

## Additional Context

nix-build output...

```
[<redac>@<redac>:~/Downloads/somewm-0.5.0]$ nix-build
this derivation will be built:
  /nix/store/lfl7zd6jq73khjj70d35islza2yf0jky-somewm-dev.drv
building '/nix/store/lfl7zd6jq73khjj70d35islza2yf0jky-somewm-dev.drv'...
Running phase: unpackPhase
unpacking source archive /nix/store/00gandfdawli582xf26rqbljrg2mfiz0-somewm-0.5.0
source root is somewm-0.5.0
Running phase: patchPhase
Running phase: updateAutotoolsGnuConfigScriptsPhase
Running phase: configurePhase
no configure script, doing nothing
Running phase: buildPhase
build flags: SHELL=/nix/store/l9k32vj2aczxw62134j1x0dsh569jz2l-bash-5.2p37/bin/bash
/nix/store/l9k32vj2aczxw62134j1x0dsh569jz2l-bash-5.2p37/bin/bash: line 1: meson: command not found
make: *** [Makefile:13: all] Error 127
error: Cannot build '/nix/store/lfl7zd6jq73khjj70d35islza2yf0jky-somewm-dev.drv'.
       Reason: builder failed with exit code 2.
       Output paths:
         /nix/store/6m0gmqz4c46rh069crgih33x0xwyxz62-somewm-dev
       Last 11 log lines:
       > Running phase: unpackPhase
       > unpacking source archive /nix/store/00gandfdawli582xf26rqbljrg2mfiz0-somewm-0.5.0
       > source root is somewm-0.5.0
       > Running phase: patchPhase
       > Running phase: updateAutotoolsGnuConfigScriptsPhase
       > Running phase: configurePhase
       > no configure script, doing nothing
       > Running phase: buildPhase
       > build flags: SHELL=/nix/store/l9k32vj2aczxw62134j1x0dsh569jz2l-bash-5.2p37/bin/bash
       > /nix/store/l9k32vj2aczxw62134j1x0dsh569jz2l-bash-5.2p37/bin/bash: line 1: meson: command not found
       > make: *** [Makefile:13: all] Error 127
       For full logs, run:
         nix log /nix/store/lfl7zd6jq73khjj70d35islza2yf0jky-somewm-dev.drv
```

meson version...
`1.7.2`

--- a/default.nix
+++ b/default.nix
@@ -18,6 +18,7 @@ pkgs.stdenv.mkDerivation {
     meson
     ninja
     pkg-config
+    wayland-scanner
     gobject-introspection  # Required for lgi typelib discovery
     makeWrapper
   ];
@@ -35,8 +36,8 @@ pkgs.stdenv.mkDerivation {
     xorg.xcbutil
     libxkbcommon
     pango
+    harfbuzz
     wayland
-    wayland-scanner
     wayland-protocols
     wlroots_0_19
     xwayland
@@ -66,6 +67,7 @@ pkgs.stdenv.mkDerivation {
       --prefix GI_TYPELIB_PATH : "${pkgs.glib.out}/lib/girepository-1.0" \
       --prefix GI_TYPELIB_PATH : "${pkgs.gtk3}/lib/girepository-1.0" \
       --prefix GI_TYPELIB_PATH : "${pkgs.gobject-introspection}/lib/girepository-1.0" \
+      --prefix GI_TYPELIB_PATH : "$GI_TYPELIB_PATH" \
       --add-flags "--search ${luaEnv}/share/lua/5.1" \
       --add-flags "--search ${luaEnv}/lib/lua/5.1"
   '';

